ultraplot/axes/shared.py

```diff
@@ -184,3 +184,40 @@ def _update_ticks(
         if kwtext_extra:
             for lab in obj.get_ticklabels():
                 lab.update(kwtext_extra)
+
+    # Override matplotlib defaults ot handle multiple axis sharing
+    def sharex(self, other):
+        return self.share_axis(which="x", other=other)
+
+    def sharey(self, other):
+        self.share_axis(which="y", other=other)
+
+    # Internal function to share axes
+    def share_axis(self, which, other):
```

Comment:
```suggestion
    # Internal function to share axes
    def _share_axis(self, which, other):
```

If this is meant to be an internal API and not public, then it should be prefixed with an underscore. Otherwise, we should remove the comment.

Reply:
Depends, I put this currently as public as I think other objects may use this imo. Internal should be rephrased to ultraplot. We could also decide to make it internal -- open to both :man_shrugging:

Reply:
matplotlib does not have this function but separate functions for x and y. I think due to this discrepancy it is better to make it internal.